commands/command_track.go

```diff
@@ -91,18 +91,17 @@ ArgsLoop:
 
 		// Generate the new / changed attrib line for merging
 		encodedArg := escapeAttrPattern(pattern)
```

Comment:
I think the way the diff has displayed is actually a bit misleading here. We want to use `escapeAttrPattern` in the case when we're not using `--filename`, since that intentionally doesn't escape glob characters, so `*.png` stays `*.png` in the `.gitattributes` file.  But when we use `--filename`, we do want to escape glob characters, so we use `escapeGlobCharacters`.  The latter just uses parts of the former because the order matters.

I'm wondering, though, if perhaps this would be easier to read as an if/else, where it would be more obvious that `escapeAttrPattern` is only used if `trackFilenameFlag` is off.  Let me rewrite it that way and see if that's easier to understand.